js/kraken.js

```diff
@@ -971,23 +971,7 @@ module.exports = class kraken extends Exchange {
         }
         return {
             'id': id,
-            'clientOrderId': clientOrderId,
             'info': response,
-            'timestamp': undefined,
-            'datetime': undefined,
-            'lastTradeTimestamp': undefined,
-            'symbol': symbol,
-            'type': type,
-            'side': side,
-            'price': price,
-            'amount': amount,
```

Comment:
The only things in question are the `price` field and the `amount` field.

Reply:
yeah, price and amount are the real problem.  If we can have them as undefined I think that's the safest pattern.  The user can always fill them in they want.

Reply:
@npomfret let's consider the `price` first.
In a market-order, the returned price will be `undefined`, as it should be. So it's not a problem.
In a limit-order, the returned price will be equal to the ordered price, which does not change over time. So, it shouldn't be a problem either, right?

Reply:
The exchange can change it if you specify at the wrong precision, also, if you trade like I do - I pass in a string, number a number, and the ccxt model expected a number for a price.  To be clear, the only time it can change is if the exchange changes the price as you place the limit order.  I've seen several exchanges do this.  It's super annoying, much better if they just error, but some will change it.  In general I think both price an amount are not safe to **assume**, but probably the other fields are ok.